docs/getting-started/1-usage/web-component.md

```diff
@@ -0,0 +1,103 @@
+# Web Component
+
+Integrating Elements into any project is a case of loading up some JavaScript and CSS, and embedding a Web Component (custom HTML) into the HTML of whatever existing documentation you have. 
+
+This will work with pretty much any CMS: Drupal, Joomla, Wordpress, Jekyll, they will all allow custom HTML to be embedded, and this approach is supported by [pretty much all modern browsers](https://caniuse.com/custom-elementsv1).
+
+## Usage
+
+1. Load the Elements web component JavaScript and default CSS. These can go in the `<head>` tag.
+  
+```html
+<script src="https://unpkg.com/@stoplight/elements@beta/web-components.min.js"></script>
+<link rel="stylesheet" href="https://unpkg.com/@stoplight/elements@beta/styles.min.css">  
+```
+
+2. Find an OpenAPI document to work with. You could create a document using [Stoplight Studio](https://stoplight.io/studio), [generate it from source code](https://stoplight.io/blog/generate-documentation-from-code/), or browse through some [real-world examples](https://apis.guru/browse-apis/).
+
+3. Embed the web component inside the HTML `<body>` tag.
+
+```html
+<elements@beta-api
+  apiDescriptionUrl="https://raw.githubusercontent.com/stoplightio/studio-demo/master/reference/todos/todo.v1.yaml"
+  router="hash"
+/>
+```
+
+Alternatively, copy and paste this into an `index.html` and open it in a browser.
+
+
+```html
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Elements in HTML</title>
+  
+    <script src="https://unpkg.com/@stoplight/elements@beta/web-components.min.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@stoplight/elements@beta/styles.min.css">
+  </head>
+  <body>
+
+    <elements-api
+      apiDescriptionUrl="tictactoe.yaml"
```

Comment:
We should a prerequisite for downloading the `tictacotoe.yaml`, as in `/hello-world.md`